### PR TITLE
add mad recipe

### PR DESCRIPTION
**Pull request by soumith**
Checklist

- [x] License file is packaged
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc: @conda-forge/help-c-cpp

### CODE REVIEW

**Comment by soumith** [comment]
this is ready for review / merge. thanks for reviewing.

File: recipes/mad/build.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+
+if [[ $(uname) == Darwin ]]; then
+    export CC=clang
```
Comment (xhochy) [inline recipes/mad/build.sh]: Why are you using `clang` instead of `$CC` as set by conda-build here?

Comment (soumith) [reply recipes/mad/build.sh]: copied an old recipe and modified it. apparently it was needed at some point, but no longer necessary.
I've deleted that line in the latest commit

**Comment by soumith** [comment]
thanks a lot @xhochy

**Pull request merged**